diff --git a/src/shebang.rs b/src/shebang.rs
index 0fc03862..f61546a4 100644
--- a/src/shebang.rs
+++ b/src/shebang.rs
@@ -17,7 +17,7 @@ impl<'line> Shebang<'line> {
       .trim()
       .splitn(2, [' ', '\t']);
 
-    let interpreter = pieces.next().unwrap_or("");
+    let interpreter = format!("@TERMUX_PREFIX@{}", pieces.next().unwrap_or(""));
     let argument = pieces.next();
 
     if interpreter.is_empty() {
